### PR TITLE
Copy Data over to get something to show....

### PR DESCRIPTION
**Pull request by Carreau**
Also add a notebook with instructions.

### CODE REVIEW

File: How to Launch RStudio.ipynb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While you can use R in this notebook, this Binder is menat to show an RStudio Demo.\n",
```
Comment (choldgraf) [inline How to Launch RStudio.ipynb]: meant!

File: Dockerfile [hunk 2]
```diff
@@ -3,3 +3,5 @@ FROM rocker/binder:3.4.2
 # Run install.r if it exists
 RUN if [ -f install.R ]; then R --quiet -f install.R; fi
 
+COPY . .
```
Comment (choldgraf) [inline Dockerfile]: Does this really work? Copying the current folder to the current folder? I thought we had to move stuff to $HOME

Comment (Carreau) [reply Dockerfile]: Works when running repo2docker locally ... I can change for $HOME.

**Comment by Carreau** [comment]
Fixed both comments. My guess is that . == $HOME in this case.

**Pull request merged**